conda/environment.yml

```diff
@@ -12,12 +12,14 @@ dependencies:
   - conda-forge::cudatoolkit
   - conda-forge::h5py=*=*mpi_openmpi*
   - conda-forge::hdf5plugin
+  - conda-forge::loguru
```

Comment:
Do we also need to add `loguru` and `tqdm` dependencies to `pyproject.toml` ?

Reply:
TL;DR: We can do, though I'm not sure if it'll do anything useful.

I don't know how others do things, but for me, I use the conda env file for all production dependencies, and I only ever `pip install` packages that are dev dependencies (like `black` and `pytest_mock`).

When I install httomo locally for development with `pip install -e`, it of course does look through `pyproject.toml` for the production dependencies and sees if any need to be installed from PyPI, but because I already have a conda env with all the production dependencies in it, nothing actually gets installed from PyPI.

From my understanding, if we would want people to be able to install httomo from PyPI, then the production  dependencies in `pyproject.toml` should match the dependencies in the conda env file. But if we aren't planning to release httomo on PyPI, then putting production dependencies in `pyproject.toml` seems redundant.

It doesn't hurt to do so, it's just a bit confusing really. Having all production dependencies in `pyproject.toml` implies we're distributing httomo on PyPI, but we're not as far as I know?

Reply:
thanks, makes sense. Let us stick with conda dependencies then.

Reply:
This did make me wonder about if we should try to distribute on PyPI in the future to make it easier for users to try it out without having conda installed...

I don't know if there's any blocker from doing so, do you know of any packages we depend on that are conda only? Maybe just httomolib/httomolibgpu/tomobar (ie all DLS-related ones)? Or is maybe astra-toolbox conda only or something?

Reply:
yes, something might be missing... supporting both is a bit of a headache :)